### PR TITLE
Created HomePage and Signup page

### PR DESCRIPTION
**Pull request by bhargaviRdy**
Completed Homepage and signup page for the watches store.
Look into 
https://vibe-store-bhargavi.netlify.app/
https://vibe-store-bhargavi.netlify.app/signup.html for changes
image

### CODE REVIEW

File: css/authentication.css [hunk 1]
```diff
@@ -24,6 +23,12 @@
     background: whitesmoke;
```
Comment (Manju-sree) [inline css/authentication.css]: use color variables here

File: css/homepage.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+.categories a{
+    position: relative;
+}
+.category--text{
+    position: absolute;
+    z-index: 2;
+    left: 0;
+    bottom: 2rem;
+    width: 100%;
+    background: #00000080;
```
Comment (Manju-sree) [inline css/homepage.css:34]: better to use again a color variable here

**Review by Manju-sree — APPROVED** [review]
apart from minor changes,overall code looks fine to merge.

File: css/homepage.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+}
+.nav-bar a{
+    font-size: 1.2rem;
+    color: var(--navbar-icons-color);
+}
+.categories{
+    padding: 1rem;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
+    gap: 1rem;
```
Comment (mariyasada) [inline css/homepage.css]: you can use repeat property here

**Review by mariyasada — APPROVED** [review]
good to merge

**Pull request merged**